themes/default/content/docs/concepts/config.md

```diff
@@ -592,3 +592,20 @@ pulumi:disable-default-providers:
   - aws
   - kubernetes
 ```
+
+### `pulumi:tags`
+
+A list of [stack tags](/docs/concepts/stack/#stack-tags) which are read by the Pulumi CLI and automatically applied on the stack at
+every `pulumi up` or `pulumi refresh` action.
```

Comment:
How does `refresh` actually work with stack tags? I tried adding a tag in Pulumi Cloud but not locally, and adding one locally but not in Pulumi Cloud, and in both cases, nothing happened when I ran `pulumi refresh`.

Reply:
@Frassle shouldn't a `refresh` take the value from Pulumi Cloud and update the stack config file if the values differ?